python-sdk/src/transfers/integrations/fivetran.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from airflow.hooks.dbapi import DbApiHook
+from transfers.datasets.base import UniversalDataset as Dataset
+from transfers.integrations.base import TransferIntegrations
+
+
+class FivetranIntegration(TransferIntegrations):
+    """
+    Class for FiveTran.
+
+    """
+
+    def __init__(self, transfer_params: dict):
+        self.transfer_params = transfer_params
+        self.transfer_mapping = {}
```

Comment:
You don't need that if you implement it already in the base class.
```suggestion
```

Reply:
Its changed now